source/lib-intro.tex

```diff
@@ -604,20 +604,20 @@
 values~(\ref{basic.fundamental}).}
 
 \pnum
-The enumerated type \term{enumerated} can be written:
+The enumerated type \placeholder{enumerated} can be written:
 
 \begin{codeblock}
-enum @\term{enumerated}@ { @\term{V0}@, @\term{V1}@, @\term{V2}@, @\term{V3}@, ..... };
+enum @\placeholder{enumerated}@ { @\placeholder{V0}@, @\placeholder{V1}@, @\placeholder{V2}@, @\placeholder{V3}@, ..... };
 
-static const @\term{enumerated C0}@ (@\term{V0}@);
-static const @\term{enumerated C1}@ (@\term{V1}@);
-static const @\term{enumerated C2}@ (@\term{V2}@);
-static const @\term{enumerated C3}@ (@\term{V3}@);
+static const @\placeholder{enumerated C0}@ (@\placeholder{V0}@);
+static const @\placeholder{enumerated C1}@ (@\placeholder{V1}@);
+static const @\placeholder{enumerated C2}@ (@\placeholder{V2}@);
+static const @\placeholder{enumerated C3}@ (@\placeholder{V3}@);
   .....
```

Comment:
Those are two distinct placeholders. Please try separating them (i.e. have two \placeholder's next to each other). Does that impair the spacing?